### ISSUE TITLE
Study techniques to normalize images

### ISSUE THREAD

**Issue by DanielCoelho112** [DanielCoelho112/localization_end_to_end#39]
All input data should be normalized before entering the neural network. 

We could use local normalization (using the information of the image), or global normalization (using the information of the entire training set)  - I just invented these names, maybe there are better names...

And also, we have different techniques to normalize the data: dividing by the max values, using the mean and std...
